src/ui/FacetSlider/FacetSlider.ts

```diff
@@ -685,7 +686,7 @@ export class FacetSlider extends Component {
 
     this.slider = this.slider
       ? this.slider
-      : new Slider(sliderDiv, _.extend({}, this.options, { dateField: this.options.dateField }), this.root);
+      : new Slider(sliderDiv, { ...this.options, ...{ dateField: this.options.dateField } } as ISliderOptions, this.root);
```

Comment:
For the `dataField`, you could just add the property rather than wrapping it in an object and using the spread operator.